readme.md

```diff
@@ -16,7 +16,9 @@ The class exposes three methods for interaction and a constructor.
 
 Initializing `IpRewrite()` object will try to rewrite the IP. If the IP is rewritten, `$_SERVER["REMOTE_ADDR"]` will be updated to reflect the end-user's IP address.
 
-`isCloudFlare();` returns `true` if the `CF_CONNECTING_IP` header is present in the request.
+`isCloudFlare();` returns `true` if the `CF_CONNECTING_IP` header is present in the request and the request originates from a CloudFlare IP.
+
+`isCloudFlareIP();` returns `true` if the request originates from a CloudFlare IP Address.
```

Comment:
Let's delete this from README since we don't want the user to get confused. They should only use `isCloudFlare()`. 

> `isCloudFlareIP();` returns `true` if the request originates from a CloudFlare IP Address.

Thanks!